docs/CodeDevelopment/prerequisites.md

```diff
@@ -26,7 +26,7 @@ the expectation is that between Project Mu and TianoCore Edk2 you could use any
 1. Download latest Git For Windows from <https://git-scm.com/download/win>
 
     ``` cmd
-    https://github.com/git-for-windows/git/releases/download/v2.37.3.windows.1/Git-2.37.3-64-bit.exe
+    https://github.com/git-for-windows/git/releases/download/v2.38.0.windows.1/Git-2.38.0-64-bit.exe
```

Comment:
It has only been 9 hours since https://github.com/microsoft/mu/pull/132 merged and the version is already being updated.

I don't think my point landed in that PR but we don't need to specify exact versions. That contradicts the previous sentence that says latest. All specifying an exact version does is create an opportunity for `latest` != `exact version here` to come and confuse people.

For example, I went to that link and can see the latest version is "2.38" right away.

![image](https://user-images.githubusercontent.com/21320094/193983866-b0ec9a84-f531-429e-98ae-c1481f632f5c.png)

Reply:
This a good point. I changed the cmd to point to the https://git-scm.com/download/win, so that the link can easily be copied.  

The same changes was applied to the python cmd.